docs/tutorials/text.rst

```diff
@@ -225,7 +225,7 @@ Dang, you know I thought that example would just work, but it looks like there a
     :width: 500
     :class: add-border
 
-Got it! If you comment out the ``.filter_contours`` line, you should see the little speck show up again.
+Got it! If you comment out the ``.filter`` line, you should see the little speck show up again.
```

Comment:
Again, kind of guessing but the above code snippet doesn't have a `.filter_contours` line, I thought maybe the lambda was originally pulled out into a function named that?

I was only skimming the docs, and not following along playing with any code myself so I didn't get too acquainted with what the code was doing. Soon!